components/main/Main.tsx

```diff
@@ -73,51 +74,62 @@ export const Main = ({ signOut, appVersion }: IMain) => {
     content = (
       <>
         <Router history={history}>
-          <PageTitle />
-          <HEEHeader signOut={signOut} />
-          <Breadcrumbs />
-          <main className="nhsuk-width-container nhsuk-u-margin-top-5">
-            <Switch>
-              <Route exact path="/home" component={Home} />
-              <Route exact path="/placements" component={Placements} />
-              <Route exact path="/programmes" component={Programmes} />
-              <Route exact path="/profile" component={Profile} />
-              <Route path="/credential" component={Dsp} />
-              <Route path="/formr-a" component={FormRPartA} />
-              <Route path="/formr-b" component={FormRPartB} />
-              <Route exact path="/support" component={Support} />
-              <Route path="/mfa" component={MFA} />
-              <Redirect exact path="/" to="/home" />
-              <Redirect
-                exact
-                path="/credential-issued"
-                to={{
-                  pathname: "/credential/issued",
-                  search: location.search
-                }}
-              />
-              <Redirect
-                exact
-                path="/credential-verified"
-                to={{
-                  pathname: "/credential/issue",
-                  search: location.search
-                }}
-              />
-              <Redirect
-                exact
-                path="/invalid-credential"
-                to={{
-                  pathname: "/credential/invalid",
-                  search: location.search
-                }}
-              />
-              <Route path="/*" component={PageNotFound} />
-            </Switch>
-          </main>
-          <HEEFooter appVersion={appVersion} />
+          <App signOut={signOut} appVersion={appVersion} />
```

Comment:
This needed to be moved to a function to allow `useLocation()` to work within the `<Router>`